### DISCUSSION TITLE
Basic help- getting a community project (Stirling PDF)

### DISCUSSION THREAD

**Discussion by ParkerS13** [community/community#129775] [New to GitHub]
Hi,
I am _very_ new to GitHub and utterly confused. I don't understand programming and I'm just here because I want to download(?) [Stirling PDF](https://github.com/Stirling-Tools/Stirling-PDF?tab=readme-ov-file). I can't figure out how to do that. I am so lost. 

Someone please help!

**Comment by Daeraxa** [comment]
On the repository page (where that link takes you) will be a section on the right hand side saying "Releases". 
![image](https://github.com/community/community/assets/58074586/33db5c27-f878-48d6-b16d-639a4472ab85)


Click the link with the little tag name and it will take you to the release page. Scroll down a bit and there will be the download links in the box marked "Assets". Just pick the download you need from there.

![image](https://github.com/community/community/assets/58074586/e14b079a-608f-40c0-8fed-1cdfd0858875)

**Reply by ParkerS13** [reply]
Okay that is very helpful, thank you. Other than, I don't know what download I need. I'm running windows 10 and I know I want their full version, but that doesn't tell me anything about what I need from the files available. 

I'm setting it up for the first time on my computer, so do I need to download all the Assets? If so where should I save them to on my computer?? And why are they not in a folder together?

This part of the read me might be helpful but i can't tell:

![StirlingDocker](https://github.com/community/community/assets/173806050/b4136c8f-4137-4f8f-8fbe-378d44195b7d)

**Reply by Daeraxa** [reply]
I'm afraid it isn't an application I'm familiar with, if you have questions about it specifically you might want to try their own discussion board - https://github.com/Stirling-Tools/Stirling-PDF/discussions

**Reply by Frooodle** [reply]
https://stirlingtools.com/docs/Getting%20started/Installation/Local/Windows%20Installation

Sorry about the confusing docs on GitHub it's catered to people which use unix or a app called docker 

The above link is our website which explains the windows side 
I'll try have this copied over to GitHub side too! Thanks 
:)

**Reply by ghostinhershell** [reply]
Thanks so much for chiming in here @Frooodle! 🚀 

Hi @ParkerS13 👋🏾,It looks like @Frooodle's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/275461a2-2a93-4887-8ba8-6a4b06e374a2)
